### PR TITLE
FIX Fix local startup instructions

### PR DESCRIPTION
**Pull request by IlliaHalchun**

### CODE REVIEW

File: front-end/README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 To run it locally, you need:
 
 1. Install the latest LTS versions of [Node.js](https://nodejs.org).
-2. Open `front-end` folder and initialize the project by running `npm i` command.
+2. Open `front-end` folder and initialize the project by running `npm i --legacy-peer-deps` command.
```
Comment (ayende) [inline front-end/README.md:11]: why is this needed?

Comment (ml054) [reply front-end/README.md:11]: old deps etc. It doesn't build w/o it.
